faq.md

```diff
@@ -62,101 +100,125 @@ Commander X16P Features & Specifications (subject to change)
   * IEC compatible (Commodore) disk drive port
   * Internal SD card slot on VERA
 * Audio
-  * Yamaha YM2151 sound chip
-  * PCM audio on VERA
-  * PSG waveform generator on VERA (simple waveforms and frequency. No envelopes or effects.)
+  * Yamaha YM2151 sound chip (8 channels of 4-Op FM)
+  * PCM audio on VERA (Up to 48kHz, 16-bit stereo)
+  * PSG of 16 channels with 4 selectable waveforms
 * "VERA" module specifications
   * Video generator featuring:
-    * Multiple output formats (VGA, NTSC Composite, NTSC S-Video, RGB video) at a fixed resolution of 640x480@60Hz
+    * Multiple output formats (VGA, NTSC Composite, NTSC S-Video, RGB video) at
+      a fixed resolution of 640x480@60Hz
     * Support for 2 layers, both supporting:
     * 1/2/4/8 bpp tile and bitmap modes
     * Support for up to 128 sprites (with inter-sprite collision detection).
   * Embedded video RAM of 128 KB.
   * Palette with 256 colors selected from a total range of 4096 colors.
-  * 16-channel stereo Programmable Sound Generator with multiple waveforms (Pulse, Sawtooth, Triangle, Noise)
-  * High quality PCM audio playback from an 4 KB FIFO buffer featuring up to 48kHz 16-bit stereo sound.
+  * 16-channel stereo Programmable Sound Generator with multiple waveforms
+    (Pulse, Sawtooth, Triangle, Noise)
+  * High quality PCM audio playback from an 4 KB FIFO buffer featuring up to
+    48kHz 16-bit stereo sound.
   * SecureDigital storage.
 * RTC: Microchip MCP7940N
 * Board power consumption: ~15W
 * PSU: TBC
 
 ## When will the X16 be released?
 
-The first batch of dev boards have been received by David. The team is currently hand soldering development boards, and they are working on a solder dip station for rapid assembly. The first 100 boards should be delivered this year, with additional sales to follow. 
+The X16 is now readily available on texelec.com for $349.99! We hope to keep it
```

Comment:
Website could be made into a clickable link.

Reply:
@tomxp411 thoughts on this? I didn't add a link since one wasn't here before but the thought had crossed my mind as well. Only thing about the link to texelec specifically is if Kevin changes the URL pattern any, it'll break and since we haven't been updating the FAQ often, that felt like a potential oops.